### PR TITLE
[kservice] implement rt_strcpy()

### PR DESCRIPTION
**Pull request by mysterywolf**
## 拉取/合并请求描述：(PR description)

[
void *rt_memmove(void *dest, const void *src, rt_ubase_t n);
rt_int32_t rt_memcmp(const void *cs, const void *ct, rt_ubase_t count);
char *rt_strncpy(char *dest, const char *src, rt_ubase_t n);
rt_int32_t rt_strncmp(const char *cs, const char *ct, rt_ubase_t count);

这四个函数
rt_ubase_t 改 rt_size_t 
以符合标准c库的类型
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [x] 本拉取/合并使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](../documentation/coding_style_cn.md) This PR complies with [RT-Thread code specification](../documentation/coding_style_en.txt)

### CODE REVIEW

File: src/kservice.c [hunk 1]
```diff
@@ -448,6 +449,21 @@ char *rt_strncpy(char *dst, const char *src, rt_ubase_t n)
...
+ *
+ * @param  dst points to the address used to store the copied content.
+ *
+ * @param  src is the string to be copied.
+ *
+ * @return The address where the copied content is stored.
+ */
+char *rt_strcpy(char *dst, const char *src)
+{
+    return rt_strncpy(dst, src, (rt_size_t)-1);
```
Comment (Guozhanxin) [inline src/kservice.c:463]: 这里写个 -1 好吗？

Comment (mysterywolf) [reply src/kservice.c:463]: 嗯，rt_sprintf就是这么写的

Comment (mysterywolf) [reply src/kservice.c:463]: 这么写没什么问题，不在意数量

Comment (ErnestChen1) [reply src/kservice.c:463]: 看着不统一

Comment (mysterywolf) [reply src/kservice.c:463]: 啥不统一

**Review by Guozhanxin — APPROVED** [review]

**Pull request merged**